Manual.md

```diff
@@ -704,6 +704,8 @@ used.
 - `archs` Whitespace separated list of architectures that a package can be
 built for, available architectures can be found under `common/cross-profiles`
 alongside the `noarch` value for packages that do not contain any machine code.
+This should only be used when upstream explicitly states that their software only works
+on specific architectures.
```

Comment:
In general, `archs` should only be set if the upstream software explicitly targets certain architectures or there is a compelling reason why the software should not be available on some supported architectures.

Reply:
Thanks, PR updated.